src/main/webapp/app/modeling-editor/modeling-editor.component.ts

```diff
@@ -55,16 +60,21 @@ export class ModelingEditorComponent implements OnInit, OnDestroy, ComponentCanD
         private route: ActivatedRoute,
         private modelingEditorService: ModelingEditorService,
         private modalService: NgbModal,
-        private translateService: TranslateService
-    ) {}
+        private translateService: TranslateService,
+        private router: Router
+    ) {
+        this.isSaving = false;
+        this.autoSaveTimer = 0;
+    }
 
     ngOnInit() {
         this.subscription = this.route.params.subscribe(params => {
             if (params['participationId']) {
                 this.modelingEditorService.get(params['participationId']).subscribe(data => {
                     this.participation = data.participation;
-                    if (data.participation.results) {
-                        this.result = data.participation.results[0];
+                    if (this.participation.results) {
+                        this.participation.results.sort((a, b) => +new Date(b.completionDate) - +new Date(a.completionDate));
```

Comment:
can we extract a function in the participation class for sorting the results? also add some documentation how they are sorted in this case